_data/tools.yml

```diff
@@ -188,6 +188,15 @@
   v2: true
   v3: true
 
+- name: RapiPdf
+  category: documentation
+  description: Custom Element to generate PDF from OpenAPI spec.
+  link: https://mrin9.github.io/RapiPdf
+  github: https://github.com/mrin9/RapiPdf
+  language: Custom Element
```

Comment:
is this a SaaS or something else?

Reply:
it is a [web component](https://developer.mozilla.org/en-US/docs/Web/Web_Components) that you can use in any of your HTML to allow your users to generate PDFs from OpenAPI spec

Here is a Demo code https://mrin9.github.io/RapiPdf/#qstart on how to do it.

You may use this link as a SaaS if you are content with  the outcome, but there are plenty of options, from changing color schemes to filter out sections of documentation, in that case you need to implement this component your self.

Reply:
bump